pxr/usd/usdSkel/CMakeLists.txt

```diff
@@ -93,6 +97,16 @@ pxr_library(usdSkel
         images/influencesPrimvarLayout.svg
         images/unboundedInterpolationExample.svg
 )
+
+pxr_build_test(testUsdSkelValidators
+        LIBRARIES
+        sdr
```

Comment:
I am not sure if sdr is needed as a dependency here. Can you remove this please? Thanks

Reply:
removed, thanks